GitUpdateScript.sh

```diff
@@ -13,7 +13,7 @@ SECONDS=0  #set seconds counter back to 0
 function GoToGitRepositoryDirectory(){
 
 	cd $home_dir${file_dir:1:-4} #change to git dir
-	pwd #pint dir to screen
+	echo "Starting git repository refresh @" $(pwd) #print dir to screen
```

Comment:
added echo before pwd to improve style of the output

Reply:
(Y)